test/Twig/Tests/Extension/IntlTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+require_once __DIR__.'/../../../../lib/Twig/Extensions/Extension/Intl.php';
+
+/**
+ * @author Remy Gazelot <r.gazelot@gmail.com>
+ */
+class Twig_Tests_Extension_IntlTest extends PHPUnit_Framework_TestCase
+{
```

Comment:
tests for this extension need to be skipped entirely if the intl extension is not loaded